Project.toml

```
@@ -52,7 +52,7 @@ ApproxManifoldProducts = "0.6.3"
 BSON = "0.2, 0.3"
 Combinatorics = "1.0"
 DataStructures = "0.16, 0.17, 0.18"
-DistributedFactorGraphs = "0.19.3, 0.20"
+DistributedFactorGraphs = "0.20"
```

```suggestion
DistributedFactorGraphs = "0.19.3 0.20"
```
Lets just leave 19 until 20 is tagged so tests can pass